### PR TITLE
Update encoded credentials for artifact publishing.

### PR DESCRIPTION
**Pull request by armiol**

### CODE REVIEW

**Comment by armiol** [comment]
@dmdashenkov PTAL.

File: scripts/publish_artifacts.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ echo " -- PUBLISHING: current branch is $TRAVIS_BRANCH."
 
 if [ $TRAVIS_BRANCH == 'master' ]; then
     echo " ------ Publishing the artifacts to the repository..."
-    openssl aes-256-cbc -K $encrypted_0f12c1faf1fc_key -iv $encrypted_0f12c1faf1fc_iv -in credentials.properties.enc -out credentials.properties -d
+    openssl aes-256-cbc -K $encrypted_81fbdd9492fa_key -iv $encrypted_81fbdd9492fa_iv -in credentials.properties.enc -out ./credentials.properties -d
```
Comment (dmdashenkov) [inline scripts/publish_artifacts.sh]: Why do we have different filepath notations here and in Spine core?

**Comment by dmdashenkov** [comment]
@armiol LGTM

File: scripts/publish_artifacts.sh (hunk 1 above)
Comment (armiol) [inline scripts/publish_artifacts.sh]: It depends on how the encoder was invoked. This time I used `./originalfile.path` notation.

I don't think it matters for the decoding execution.

**Pull request merged**